### PR TITLE
Move the testbed federation config to the production repositories

### PR DESCRIPTION
**Pull request by JuanPTM**
Part of https://github.com/SovereignCloudStack/issues/issues/412

### CODE REVIEW

File: all/099-kolla.yml [hunk 1]
```diff
@@ -48,6 +48,10 @@ horizon_backend_database: "yes"
 # keystone
 keystone_token_provider: "fernet"
 keystone_federation_oidc_response_type: "code"   # NOTE: use authorization code in keystone instead of default upstream
+keystone_oidc_forward_header: "X-Forwarded-Proto"
+
+enable_keystone_federation: "no"        # NOTE: Not enabled by default
```
Comment (berendt) [inline all/099-kolla.yml:53]: Comment can be removed.

File: all/099-kolla.yml [hunk 2]
```diff
@@ -48,6 +48,10 @@ horizon_backend_database: "yes"
...
+keystone_enable_federation_openid: "no"  # NOTE: Not enabled by default
```
Comment (berendt) [inline all/099-kolla.yml:54]: Comment can break removed.

File: all/099-kolla.yml [hunk 3]
```diff
@@ -48,6 +48,10 @@ horizon_backend_database: "yes"
 # keystone
 keystone_token_provider: "fernet"
 keystone_federation_oidc_response_type: "code"   # NOTE: use authorization code in keystone instead of default upstream
```
Comment (berendt) [inline all/099-kolla.yml:50]: Add this comment above the parameter.

**Review by berendt — CHANGES_REQUESTED** [review]

**Comment by JuanPTM** [comment]
Removed comments and moved the other one on  top of the parameter

**Review by berendt — APPROVED** [review]

**Pull request merged**